{"name": "Попытка входа с невалидными данными: неверный логин", "status": "passed", "steps": [{"name": "Подготовка тестовых данных для сценария: неверный логин", "status": "passed", "start": 1758210125858, "stop": 1758210125858}, {"name": "Отправляем POST-запрос для авторизации курьера", "status": "passed", "start": 1758210125858, "stop": 1758210126371}, {"name": "Проверяем код ответа 404", "status": "passed", "start": 1758210126371, "stop": 1758210126371}, {"name": "Проверяем сообщение об ошибке", "status": "passed", "start": 1758210126371, "stop": 1758210126371}], "parameters": [{"name": "test_case", "value": "'неверный логин'"}, {"name": "credentials", "value": "<function TestCourierLogin.<lambda> at 0x0000025C0D73CB80>"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1758210125858, "stop": 1758210126372, "uuid": "11a90dde-58b4-4efc-86d7-c6605a35f752", "historyId": "9fc685cf20a01c0e93588c5898a5e878", "testCaseId": "505bba1c9113ea857d47e178537c8295", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_negative_cases", "labels": [{"name": "story", "value": "Неуспешная авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "What-Is-Lowe"}, {"name": "thread", "value": "5532-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}